Virtusize/Sources/VirtusizeRepository.swift

```diff
@@ -284,7 +284,7 @@ internal class VirtusizeRepository: NSObject {
 		guard let recommendedSize = recommendedSize else {
 			return
 		}
-        bodyProfileRecommendedSizes?.append(BodyProfileRecommendedSize(sizeName: recommendedSize))
```

Comment:
Check lines 274 and 276, the size recommendation always depended on the first item of bodyProfileRecommendedSizes. 
I'm not sure why the logic was modified like this, but we only need the latest recommended size.

Reply:
> the new data was appended to bodyProfileRecommendedSizes

I see.Now I understand. It was actually appended instead of replaced. Thanks.